src/Sema.zig

```diff
@@ -18410,8 +18410,8 @@ fn zirBoolToInt(sema: *Sema, block: *Block, inst: Zir.Inst.Index) CompileError!A
     const operand = try sema.resolveInst(inst_data.operand);
     if (try sema.resolveMaybeUndefVal(operand)) |val| {
         if (val.isUndef()) return sema.addConstUndef(Type.initTag(.u1));
-        const bool_ints = [2]Air.Inst.Ref{ .zero, .one };
-        return bool_ints[@boolToInt(val.toBool())];
+        if (val.toBool()) return sema.addConstant(Type.initTag(.u1), Value.initTag(.one));
+        return sema.addConstant(Type.initTag(.u1), Value.initTag(.zero));
```

Comment:
Github won't let me commit that suggestion, trying again
```suggestion
        if (val.toBool()) return sema.addConstant(Type.u1, Value.one);
        return sema.addConstant(Type.u1, Value.zero);
```